src/main/java/com/limechain/cli/Cli.java

```diff
@@ -49,8 +50,9 @@ public CliArguments parseArgs(String[] args) {
             String network = cmd.getOptionValue("network", "").toLowerCase();
             if (!validChains.contains(network) && !network.isEmpty()) throw new ParseException("Invalid network");
             String dbPath = cmd.getOptionValue("db-path", DBInitializer.DEFAULT_DIRECTORY);
```

Comment:
`network` and `db-path` strings should be replaced with the newly extracted variables